_posts/2013-01-14-alt-text.md

```diff
@@ -1,21 +1,29 @@
+The `alt` attribute is meant to provide "alternate text" descriptions of images to help ensure people do not miss out on information conveyed by graphics.
 
-A few tips on how and when to use the `alt` attribute:
+Alternative text can help people using assistive technology, such as screen readers, or people who have images turned off on their mobile devices (to save data, for instance). Additionally, alt text can be useful to everyone in instances of poor Internet service, or your CDN of choice having a service hiccup (or attack), and images are slow or fail to load.
 
-- Use the `alt` attribute for any image that is used as content. `alt` text describes the function of an image and is rarely a detailed description of the image itself (unless, for example, the page is actually about a specific picture).
-- Make sure the description of the image is useful. For example, if the image is your logo your `alt` should be your company name and not "logo".
-- Don't begin `alt` text with "photo of.." or "picture of.."; assistive technologies already do this. Redundancy and repetition are almost as bad as not enough information.
-- [Don't put line breaks in `alt` text.](https://developer.paciellogroup.com/blog/2015/09/short-note-on-coding-alt-text/) It causes suboptimal effects when read by screen readers.
-- Using an image file several times in a website doesn't mean that the `alt` attribute has to be the same in each `img` tag, because `alt` attributes change with context. For example, as seen above, a logo that is a link to your home page should have `alt="home page"` and the same logo, used somewhere else as decoration, should have `alt=""`.
-- Use an empty `alt` attribute for any image that is decorative or not necessary for understanding the content of the page (`alt=""` or just `alt` - e.g. `<img src="prettyswirl.jpg" alt>`). If, for example, the information in a graph is explained in text below it, there's no need to duplicate that in `alt` text.
-- Images that are links should describe where they go. For example, a logo that is a link to your home page should have `alt="home page"`. But if the image is only one part of the contents of a link and the other contents (e.g. plain text) describes the link then use an empty `alt` attribute for the image - if you were to specify 'alt' text as well this would be redundant/repetitive.
+## A few tips on how and when to use the `alt` attribute:
+- Use the `alt` attribute for images that are used as more than decoration. The alternative text describes the primary purpose of an image and is rarely a detailed description of the image itself (unless, for example, the page is actually about a specific image).
+- Make sure the description of the image is useful. For example, if the image is of a company's logo, the `alt` should (often) be the company's name. The word "logo" is not necessary or useful as part of the alternative text.
```

Comment:
I think "(often)" here adds more complexity/confusion. What are the circumstances where you shouldn't when it comes to logos? How will I know if I should or should not?

Reply:
'often' was used in reference to the logo being referenced differently in other examples.  but yeh, will remove for simplicity.